init.fish

```diff
@@ -41,16 +41,20 @@ source $OMF_CONFIG/before.init.fish ^/dev/null
 set -l user_function_path $fish_function_path[1]
 set fish_function_path[1] $OMF_PATH/lib
 
-set -l theme {$OMF_PATH,$OMF_CONFIG}/themes/(cat $OMF_CONFIG/theme)
+# Autoload util functions
+autoload $OMF_PATH/lib $OMF_PATH/lib/git
 
-for path in $OMF_PATH/lib $OMF_PATH/lib/git {$OMF_PATH,$OMF_CONFIG}/pkg/* $theme
-  contains -- (basename $path) $OMF_IGNORE; and continue
+for path in {$OMF_PATH,$OMF_CONFIG}/pkg/*
+  set -l name (basename $path)
 
-  autoload $path $path/completions
-  source $path/(basename $path).fish ^/dev/null
-    and emit init_(basename $path) $path
+  contains -- $name $OMF_IGNORE; and continue
+  require $name
 end
 
+# Autoload theme
+autoload {$OMF_PATH,$OMF_CONFIG}/themes/(cat $OMF_CONFIG/theme)
```

Comment:
![feature](https://www.vardot.com/sites/default/files/brnfdxpimaaaytn_1.jpg)

The idea is to support overwriting specific functions on your theme/plugin. So if you correctly split functions on individual files, one have the opportunity to change a specific part of it.